packaging/windows/buildpackage.sh

```diff
@@ -64,39 +63,27 @@ function makelauncher()
 	wine64 rcedit-x64.exe "${BUILTDIR}/${LAUNCHER_NAME}.exe" --set-icon "${BUILTDIR}/${MOD_ID}.ico"
 }
 
-function build_platform()
-{
-	PLATFORM="${1}"
-
-	echo "Building core files (${PLATFORM})"
-	if [ "${PLATFORM}" = "x86" ]; then
-		USE_PROGRAMFILES32="-DUSE_PROGRAMFILES32=true"
-	else
-		USE_PROGRAMFILES32=""
-	fi
-
-	install_assemblies "${SRCDIR}" "${BUILTDIR}" "win-${PLATFORM}" "net6" "False" "True" "True"
-	install_data "${SRCDIR}" "${BUILTDIR}" "cnc" "d2k" "ra"
-	set_engine_version "${TAG}" "${BUILTDIR}"
-	set_mod_version "${TAG}" "${BUILTDIR}/mods/cnc/mod.yaml" "${BUILTDIR}/mods/d2k/mod.yaml" "${BUILTDIR}/mods/ra/mod.yaml" "${BUILTDIR}/mods/cnc-content/mod.yaml" "${BUILTDIR}/mods/d2k-content/mod.yaml" "${BUILTDIR}/mods/ra-content/mod.yaml"
-
-	echo "Compiling Windows launchers (${PLATFORM})"
-	makelauncher "RedAlert" "Red Alert" "ra" "${PLATFORM}"
-	makelauncher "TiberianDawn" "Tiberian Dawn" "cnc" "${PLATFORM}"
-	makelauncher "Dune2000" "Dune 2000" "d2k" "${PLATFORM}"
-
-	echo "Building Windows setup.exe ($1)"
-	makensis -V2 -DSRCDIR="${BUILTDIR}" -DTAG="${TAG}" -DSUFFIX="${SUFFIX}" -DOUTFILE="${OUTPUTDIR}/OpenRA-${TAG}-${PLATFORM}.exe" ${USE_PROGRAMFILES32} OpenRA.nsi
-
-	echo "Packaging zip archive ($1)"
-	pushd "${BUILTDIR}" > /dev/null
-	zip "OpenRA-${TAG}-${PLATFORM}-winportable.zip" -r -9 ./* --quiet
-	mv "OpenRA-${TAG}-${PLATFORM}-winportable.zip" "${OUTPUTDIR}"
-	popd > /dev/null
-
-	rm -rf "${BUILTDIR}"
-}
+echo "Building core files"
+
+install_assemblies "${SRCDIR}" "${BUILTDIR}" "win-x64" "False" "True" "True"
+install_data "${SRCDIR}" "${BUILTDIR}" "cnc" "d2k" "ra"
+set_engine_version "${TAG}" "${BUILTDIR}"
+set_mod_version "${TAG}" "${BUILTDIR}/mods/cnc/mod.yaml" "${BUILTDIR}/mods/d2k/mod.yaml" "${BUILTDIR}/mods/ra/mod.yaml" "${BUILTDIR}/mods/cnc-content/mod.yaml" "${BUILTDIR}/mods/d2k-content/mod.yaml" "${BUILTDIR}/mods/ra-content/mod.yaml"
+
+echo "Compiling Windows launchers"
+makelauncher "RedAlert" "Red Alert" "ra"
+makelauncher "TiberianDawn" "Tiberian Dawn" "cnc"
+makelauncher "Dune2000" "Dune 2000" "d2k"
+
+echo "Building Windows setup.exe"
+makensis -V2 -DSRCDIR="${BUILTDIR}" -DTAG="${TAG}" -DSUFFIX="${SUFFIX}" -DOUTFILE="${OUTPUTDIR}/OpenRA-${TAG}-${PLATFORM}.exe" OpenRA.nsi
```

Comment:
Missed a platform &rarr; x64 here. Do we even want/need to include the x64 in these filenames now?